security/src/main/java/com/networknt/security/JwtVerifier.java

```diff
@@ -214,19 +223,19 @@ public JwtClaims verifyJwt(String jwt, boolean ignoreExpiry, boolean isToken) th
      * token in the request header to renew the expired token.
      *
      * @param jwt String of Json web token
-     * @param ignoreExpiry If true, don't verify if the token is expired.
+     * //@param ignoreExpiry If true, don't verify if the token is expired.
```

Comment:
Please examine if removal of ignoreExpiry in signature will cause a backwards comapatibility issue.

In general, if you remove an argument, please fully remove its parameter documentation, do not simply use //